mpas_analysis/ocean/time_series_ohc.py

```diff
@@ -591,6 +634,20 @@ def run(self):  # {{{
                                          lineStyles=['r-', 'r-', 'r--', 'r-.'],
                                          lineWidths=[2, 1, 1.5, 1.5],
                                          calendar=calendar)
+
+                caption = 'Running Mean of {} Ocean Heat Content'.format(region)
+                write_image_xml(
+                    config=config,
+                    filePrefix=self.filePrefixes['OHC', region],
+                    componentName='Ocean',
+                    componentSubdirectory='ocean',
+                    galleryGroup='Time Series',
+                    groupLink='timeseries',
+                    thumbnailDescription=u'{} ΔOHC'.format(region),
```

Comment:
Not sure if this should be a Delta or not  but the thumbnail descriptions should be unique for each thumbnail so maybe this should be `'running mean {} OHC'`?  It's a little long but  I think that's okay.